MINT.postman_collection.json

```diff
@@ -298,7 +298,7 @@
+								"query": "mutation updatePlanBeneficiaries{\n    updatePlanBeneficiaries (id: \"{{beneficiariesID}}\", changes: \n        {\n            status: READY_FOR_CLEARANCE\n        #Page 1\n        beneficiaries: [MEDICAID,MEDICARE_PART_D]\n        diseaseSpecificGroup: \"centers for diabetes prevention\"\n        #   beneficiariesOther:\n        #   beneficiariesNote:\n        #   treatDualElligibleDifferent:\n        #   treatDualElligibleDifferentHow:\n        #   treatDualElligibleDifferentNote:\n        #   excludeCertainCharacteristics:\n        #   excludeCertainCharacteristicsCriteria:\n        #   excludeCertainCharacteristicsNote:\n        #Page 2\n        numberPeopleImpacted: 22\n        #   estimateConfidence:\n        confidenceNote: \"Confidence hooray!!!!\"\n        #   beneficiarySelectionMethod:\n        #   beneficiarySelectionOther:\n\n        #   beneficiarySelectionNote:\n        #   #Page 3\n        #   beneficiarySelectionFrequency:\n        #   beneficiarySelectionFrequencyOther:\n        #   beneficiarySelectionFrequencyNote:\n        #   beneficiaryOverlap:\n        #   beneficiaryOverlapNote:\n        #   precedenceRules: \n        #   precedenceRulesYes: \n        #   precedenceRulesNo:\n        #   precedenceRulesNote:\n    }) {\n        id\n        modelPlanID\n        beneficiaries\n        beneficiariesOther\n        beneficiariesNote\n        diseaseSpecificGroup\n        treatDualElligibleDifferent\n        treatDualElligibleDifferentHow\n        treatDualElligibleDifferentNote\n        excludeCertainCharacteristics\n        excludeCertainCharacteristicsCriteria\n        excludeCertainCharacteristicsNote\n        numberPeopleImpacted\n        estimateConfidence\n        confidenceNote\n        beneficiarySelectionMethod\n        beneficiarySelectionOther\n        beneficiarySelectionNote\n        beneficiarySelectionFrequency\n        beneficiarySelectionFrequencyOther\n        beneficiarySelectionFrequencyNote\n        beneficiaryOverlap\n        beneficiaryOverlapNote\n        precedenceRules\n        precedenceRulesYes\n        precedenceRulesNo\n        precedenceRulesNote\n        createdBy\n        createdDts\n        modifiedBy\n        modifiedDts\n        readyForReviewBy\n        readyForReviewDts\n        readyForClearanceBy\n        readyForClearanceDts\n        status\n\n        createdByUserAccount \n        {\n            commonName\n        }   \n        modifiedByUserAccount\n        {\n            commonName\n        }     \n        readyForReviewByUserAccount \n        {\n            commonName\n        }\n        readyForClearanceByUserAccount \n        {\n            commonName\n        }        \n    }\n}",
```

Comment:
Thanks for updating the return type. [Original Conversation](https://github.com/CMSgov/mint-app/pull/842#discussion_r1422890111) Can you also add on setting the values? It would be helpful to see that the values are set when you run the mutation.
```GQL
       precedenceRules: [YES,NO]
        precedenceRulesYes: "It is a yes"
        precedenceRulesNo: "It is a no"
        precedenceRulesNote: "Precedence notes go here"
```

Reply:
Ah yes, I intended to migrate this talk to the open PR. Thanks for bringing this up.

We have a large variety of variables which do not have test values set. It is unclear to me which is the process we want standard, so I went with the majority of settings. Is there a reason we want values for these variables set but not others?

Reply:
Thanks Tom, 

My thoughts here, it would be ideal to have values set for every field, but the postman scripts where largely first committed not filled out. It's not the biggest gain to do a tech debt ticket to fill out every field.

However, I think there is some value if you have to modify the postman script already to include the new values. This has the benefit of allowing a reviewer to verify that the new fields work, and have data filled out. Otherwise, we just have to do it every time we want to test.

 So I figure if we have a situation where we can easily add the new fields, it's ideal to improve it a little as we go. (And makes validating even easier)

Reply:
That makes sense. Since I understand now that we consider the fields without variables to be tech debt, I'll make a point in the future to leave the value assignments.

Reply:
thanks Tom, I appreciate it. It seems like fields we are likely updating for our testing already, so it doesn't seem like we will have to do much more than we currently are.